app/suggestion/views.py

```diff
@@ -1,16 +1,115 @@
-from flask import render_template
+from datetime import datetime
+from flask import abort, flash, render_template, redirect, url_for
 from flask.ext.login import login_required
+import pytz
```

Comment:
nit: move up

format we use:
python's native libs
newline
external libs
newline
app's own libs